runtime/Go/antlr/atn.go

```diff
@@ -4,8 +4,11 @@
 
 package antlr
 
+// ATNInvalidAltNumber represents an invalid ATN alternative
```

Comment:
Thanks for filling out the doc, that's good. To fit Go convention, doc comments should be full sentences: https://golang.org/doc/effective_go#commentary. This one needs ending punctuation.